packages/cardano-services-client/src/version.ts

```diff
@@ -1,5 +1,2 @@
-// auto-generated using /home/rhys/code/iohk/cardano-js-sdk/packages/cardano-services-client/scripts/createVersionSource.js
-export const version = {
-  api: '1.0.0',
-  software: '0.9.9'
-};
+// auto-generated using ../scripts/createVersionSource.js
+export const apiVersion = '1.0.0';
```

Comment:
Since we're using package.json as source for software version, perhaps it would be better to patch package.json and have versions contained in 1 file?

Reply:
Let's do this in the next iteration? I'll need to update this implementation to consider all provider versions.